builtin/settingtypes.txt

```diff
@@ -623,6 +623,13 @@ update_last_checked (Last update check) string
 #    Ex: 5.5.0 is 005005000
 update_last_known (Last known version update) int 0
 
+#    Type of occlusion_culler
+occlusion_culler (Occlusion Culler) enum old old,new
+
+#    Use raytraced occlusion culling in the new culler.
+#	 This flag enables use of raytraced occlusion culling test
+enable_raytraced_culling (Enable Raytraced Culling) bool true
```

Comment:
What are the benefits and disadvantages of this? What are the usecases of disabling it?

Reply:
Without the extra (old) raytraced culler there are fewer blocks filtered. In some scenes that can be dramatic (like 10x as many blocks). We can not yet delete the old culler.

I support the extra setting, though, as eventually we would want to remove the old raytraced culler. 
One thing I am keen about is to not let the server and clients cullers drift too far apart. Hence my desire to put this into Map (instead of clientmap) and it this to the server culler as well.

Reply:
This setting allows for testing more of the new culler while in beta-testing. We can remove the setting when removing the old traversing algorithm.